focus-areas/security/openssf-best-practices.md

```diff
@@ -1,41 +1,67 @@
+- Advanced badge levels of silver and gold are available if the project satisfies the additional criterion.
+
+For further information, refer to [OpenSSF’s API documentation](https://bestpractices.coreinfrastructure.org/en) 
+
+### Filters
+The metric can be filtered by:
+- Badge level (Passing, Silver, Gold)
+- Project type or focus area (e.g., cybersecurity projects)
+- Criteria status (Met, Unmet, N/A, Unknown)
+- Compliance trends over time
 
 ### Visualizations
-
 ![OpenSSF visualizations](https://raw.githubusercontent.com/chaoss/wg-risk/main/focus-areas/security/images/cii-best-practices_visualization.png)
+   *Figure 1: Example of OpenSSF Best Practices Badge (OpenSSF)* 
 
-### Tools Providing Metric
-
-Augur provides an example implementation for the OpenSSF Best Practices metric.
-An example of OpenSSF metrics in use can be found at http://augur.osshealth.io/repo/Zephyr-RTOS/zephyr/risk
-
-### Data Collection Strategies
-
-See [OpenSSF’s API documentation](https://github.com/coreinfrastructure/best-practices-badge/blob/master/doc/api.md) for relevant information.
-
-As from the [OpenSSF Best Practices Badging Page](https://bestpractices.coreinfrastructure.org/en): Projects receive the passing badge if they meet all of the required criteria. The status of each criterion, for a given project, can be 'Met', 'Unmet', 'N/A' or 'unknown'. Each criterion is in one of four categories: 'MUST', 'SHOULD', 'SUGGESTED', or 'FUTURE'. To obtain a badge, all the MUST and MUST NOT criteria must be met, all SHOULD criteria must be met OR the rationale for not implementing the criterion must be documented, and all SUGGESTED criteria have to be rated as met or unmet. Advanced badge levels of silver and gold are available if the project satisfies the additional criterion. 
+</details></span>
 
 ## References
-
-- OpenSSF Badging Website: https://bestpractices.coreinfrastructure.org/en 
-- Augur: https://github.com/chaoss/augur
-
-
-
+- [OpenSSF Badging Website](https://bestpractices.coreinfrastructure.org/en)
+- [Augur](https://github.com/chaoss/augur)
+
+## Contributors
+- Elizabeth Barron
+- Sean Goggins
+- Matt Germonprez
+- Daniel Izquierdo
+- Dawn Foster
+- Beth Hancock
+- Kevin Lumbard
+- Vinod Ahuja
+- Yigakpoa L. Samuel (Ikpae)
+
+## Additional Information
+*The usage and dissemination of health metrics may lead to privacy violations. Organizations may be exposed to risks. These risks may flow from compliance with the GDPR in the EU, with state law in the US, or with other law. There may also be contractual risks flowing from terms of service for data providers such as GitHub and GitLab. The usage of metrics must be examined for risk and potential data ethics problems. Please see [CHAOSS Data Ethics document](https://github.com/chaoss/community/blob/main/data-use-statement.md) for additional guidance, and [OpenSSF’s API documentation](https://github.com/coreinfrastructure/best-practices-badge/blob/master/doc/api.md) for relevant information.*

```

Comment:
We actually add this as a separate module in the Wordpress page so you can delete this paragraph entirely. Thanks!

Reply:
Done, thank you